libs/assets/src/lib/asset-details-dialog.tsx

```diff
@@ -183,10 +204,25 @@ export const AssetDetailsDialog = ({
       icon={<Icon name="info-sign"></Icon>}
       open={open}
       onChange={(isOpen) => onChange(isOpen)}
+      onCloseAutoFocus={(e) => {
+        /**
+         * This mimics radix's default behaviour that focuses the dialog's
+         * trigger after closing itself
+         */
+        if (trigger) {
```

Comment:
question (blocking): why do we need to recreate radix's behaviour here? Can we not rely on that?

Reply:
We can't because we don't use radix's trigger 😞 for the asset details dialog.